src/Maintenance/Maintenance/ChangeLog.md

```diff
@@ -20,6 +20,8 @@
 
 ## Upcoming Release
 * Fixed bug where rebootSettings property wasn't updating.
+* Fixed bug where Get-MaintenanceConfig was returning a List.
```

Comment:
Please always insert new change logs between L21-L22 to ensure new change logs is under the top of upcoming release section after merging into main branch

Reply:
Updated this in the next commit.